nbodykit/tests/test_bianchi.py

```diff
@@ -0,0 +1,57 @@
+from mpi4py_test import MPITest
+from nbodykit.lab import *
+from nbodykit import setup_logging
+from numpy.testing import assert_allclose
+
+# debug logging
+setup_logging("debug")
+
+@MPITest([1, 4])
+def test_bianchi(comm):
+    
+    NDATA = 1000
+    NBAR = 1e-4
+    
+    CurrentMPIComm.set(comm)
+    cosmo = cosmology.Planck15
+    
+    data = Source.RandomParticles(NDATA, seed=42)
+    randoms = Source.RandomParticles(NDATA*10, seed=84)
+    
+    # add the random columns
+    for s in [data, randoms]:
```

Comment:
What about generate random seeds for every 100,000 particles, then each rank generates only the overlapped chunks, and keeps only overlapping particles?